Source/Scene/FrameState.js

```diff
@@ -156,6 +156,12 @@ define([
              */
             sse : undefined
         };
+
+        /**
+        * A scalar used to exaggerate the terrain.
+        * @type {Number}
+        */
+        this.terrainExaggeration = undefined;
```

Comment:
Is `1.0` a better default since, well, that is the default and it is more explicit to the JavaScript engine that the type is a number?